packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -90,6 +89,21 @@ const accessManagerMenuItem = css`
   margin-bottom: 0px;
 `;
 
+const onPremMenuWrapper = css`
+  display: inline-block;
+  position: relative;
+  z-index: 0;
```

Comment:
I think I mentioned this generally, but we want. to be really careful about the selection of our z-index values. This will be a lower z-index than the fixed header itself by default. I am not 100% sure how that will turn out as a result.

Reply:
Going to do a big zindex sweep per your previous ticket, dw!

Reply:
@seanmhanson `z-index: 0` is only different than `z-index: auto` (the browser default) in that it forms a new stacking context. Largely, this does almost nothing aside from allowing us to safely use z-index on the element's descendants. We do this frequently for things like our interaction ring pattern where we rely on z-index to render it properly, but don't want elements to appear above things like modals.

Reply:
Codepen to demonstrate: https://codepen.io/mdbdave/pen/PoqWRGN

If you un-comment the `z-index` rule on `.context-parent`, it should demonstrate the effect `z-index: 0;` has on its children.

Reply:
I didn't know this! In that case 0 here sounds fine.